docs/CHANGELOG.asciidoc

```diff
@@ -93,6 +94,279 @@ you make the necessary updates after you upgrade to 8.15.0.
+* Adds an option to mark AI Assistant Knowledge Base entries as public ({kibana-pull}184094[#184094]).
+* Updates the AI assistant to query all search connectors by default and adds a setting to override this new default ({kibana-pull}183712[#183712]).
+* Adds downstream dependency service name to logs and errors to improve alert insights ({kibana-pull}183215[#183215]).
+Operations::
+* Adds password support to the Kibana keystore ({kibana-pull}180414[#180414]).
+Platform::
+* Adds http2 support to the Kibana server, that can be enabled using the `server.protocol: http2` Kibana setting ({kibana-pull}183465[#183465]).
+* Kibana's `rolling-file` appender now supports more advanced retention policies. Refer to the [Kibana logging documentation](https://www.elastic.co/guide/en/kibana/current/logging-configuration.html#rolling-file-appender) for more details ({kibana-pull}182346[#182346]).
+Security::
+* Adds an optional role description field to roles ({kibana-pull}183145[#183145]).
+* Adds the ability to filter audit logs by username using the `xpack.security.audit.ignore_filters.users` configuration setting ({kibana-pull}183137[#183137]).
+* Adds support for `remote_cluster` privileges in ES role definition ({kibana-pull}182377[#182377]).
+* Improves the experience for managing a larger number of API keys by adding server side filtering, pagination and querying. ({kibana-pull}168970[#168970]).
+
+[float]
+[[fixes-v8.15.0]]
+=== Bug Fixes
+Alerting::
+* Fixes kibana.alert.rule.execution.timestamp timezone and format ({kibana-pull}183905[#183905]).
+* Fixes undefined error source in alerting log tags ({kibana-pull}182352[#182352]).
+* Allow the rule types to throw user errors ({kibana-pull}184213[#184213]).
+* Sets validation errors in subaction framework as user errors ({kibana-pull}184317[#184317]).
+* Fixes x-axis time zone on alertSummaryWidget full size ({kibana-pull}187468[#187468]).
+Dashboards::
+* Prevent jumping control drag handle between view modes ({kibana-pull}184533[#184533]).
+* Fixes error on navigation when invalid selections tour step is open ({kibana-pull}189449[#189449]).
+* Fixes positioning of dragged link in Links editor ({kibana-pull}189122[#189122]).
+* Don't close the flyout when canceling the Save to library action ({kibana-pull}188995[#188995]).
+* Fixes error thrown on numeric options list ({kibana-pull}188789[#188789]).
+* Adds tooltip support to `PresentationPanel` header badges ({kibana-pull}186102[#186102]).
+* Fixes unsaved changes on new dashboards bug ({kibana-pull}184955[#184955]).
+* Reset `maximizedPanelId` on Dashboard navigation ({kibana-pull}183060[#183060]).
+Data Discovery::
+* Fixes time range filter ({kibana-pull}187010[#187010]).
+* Reset selected fields when modifying the ES|QL query ({kibana-pull}185997[#185997]).
+* Fixes document comparison mode and the field statistics tab when using Smart Fields ({kibana-pull}184172[#184172]).
+Discover::
```

Comment:
We have 2 Discover sections here. I think this is not necessary. Also instead of Data Discovery let's use Discover here

Reply:
Do you mean merging the Discover sections from bug fixes and enhancements together, or merging Data Discovery and Discover in each of these sections?

Reply:
(I assumed the latter in my latest commit)

Reply:
The latter, thank you!